spec/support/vcr_cassettes/lighthouse/benefits_claims/index/200_response.yml

```diff
@@ -2,7 +2,7 @@
 http_interactions:
 - request:
     method: get
-    uri: https://sandbox-api.va.gov/services/claims/v2/veterans/123498767V234859/claims
+    uri: https://example.com
```

Comment:
Please mask the URI in [support/vcr.rb](https://github.com/department-of-veterans-affairs/vets-api/blob/master/spec/support/vcr.rb)

Reply:
@RachalCassity updated